AsyncDisplayKit/TextKit/ASTextKitAttributes.h

```diff
@@ -24,7 +24,7 @@ extern NSString *const ASTextKitEntityAttributeName;
 
 static inline BOOL _objectsEqual(id<NSObject> obj1, id<NSObject> obj2)
```

Comment:
Can you just delete this function and call ASObjectIsEqual?  The latter is newer, but has enjoyed widespread adoption, so that would be great.